src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4990,6 +4990,19 @@ private IDisposable TryGenerateDeferedLoadStrategy(IIndentedStringBuilder writer
 									BuildComplexPropertyValue(innerWriter, def, closureName + ".", closureName);
 								}
 
+								if (nameMember != null)
```

Comment:
Can you add a unit test addressing the change here?

Reply:
This is tested by the runtime test I've added for the `ScrollViewer`. (Will however see if there is a simple way to validate that more closely, but I would rather prefer to merge this PR as fast as possible)

Reply:
Finally this has been fixed by https://github.com/unoplatform/uno/pull/4333, so it's no longer relevant